internal/lifecycle/lifecycle.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Google LLC
```

Comment:
Use git mv for moving files-- makes it much easier to review than if it looks like this file is entirely new.